resolver_test.go

```diff
@@ -8,7 +8,8 @@ import (
 
 func init() {
 	// Silence the logger while running tests
-	Log.SetLevel(0)
+	// TODO:fix
```

Comment:
Implementing this should be quite straightforward. Either replace the logger with one writing to io.Discard or set the level so high that it's above "error" level